skbio/diversity/tests/test_driver.py

```diff
@@ -298,20 +298,20 @@ def test_invalid_input(self):
 
         # additional kwargs
         self.assertRaises(TypeError, beta_diversity, 'euclidean',
-                          [[0, 1, 3, 4], [0, 3, 12, 42]],
+                          [[0, 1, 3], [0, 3, 12]],
```

Comment:
Noticed that these values were incorrect before, but the ``TypeError`` was getting thrown due to the extra kwarg before the ``MissingNodeError``. (The test was testing the correct thing, it just would fail for a different reason if the ``TypeError`` wasn't thrown.)